scripts/install_custom_dc.sh

```diff
@@ -83,6 +85,26 @@ terraform init && terraform apply \
   -var="project_id=$PROJECT_ID" \
   -var="dc_website_domain=$DOMAIN" -auto-approve
 
+# Run the BT automation Terraform script to set up BT loader.
+cd $ROOT
+git clone https://github.com/datacommonsorg/tools
+
+# TODO(alex): support custom robot SA and resource bucket name.
+WEBSITE_ROBOT="website-robot@$PROJECT_ID.google.com.iam.gserviceaccount.com"
+RESOURCE_BUCKET="$PROJECT_ID-resources"
+
+cd tools/bigtable_automation/terraform
+terraform init && terraform apply \
+  -var="project_id=$PROJECT_ID" \
+  -var="service_account_email=$WEBSITE_ROBOT" \
+  -var="dc_resource_bucket=$RESOURCE_BUCKET" \
+  -auto-approve
+
+# Copy over sample tmcfs/csvs from reference resource bucket.
+gsutil cp -r \
```

Comment:
gsutil cp does not seem to work..

Reply:
ack. If you mean entire folder copied over as opposed to files, folder levels are being copied over. As for the data, I will populate it shortly.